CHANGELOG.rst

```diff
@@ -22,6 +22,7 @@ Changed
 ~~~~~~~
 
 * Changed the ``inquiries`` API path from ``/exp`` to ``/api/v1`` #4495
+* Moved the lock from concurrency policies into the scheduler. #4481 (bug fix)
```

Comment:
Please also clarify in the changelog entry what bug it fixes. Otherwise if people go over the changelog they will have no idea what this change doesn't and if it affects them or not.

Reply:
Fixed